### PR TITLE
Feature/change setup arguments

### PR DESCRIPTION
**Pull request by DaveAtkinsonXMOS**
Changes to setup.sh arguments for unified AVS setup.
Also refactor some code.

### CODE REVIEW

**Review by lucianomartin — COMMENTED** [review]
Change look very good.
If they are complete, can you update 
https://github.com/xmos/vocalfusion-rpi-setup/blob/master/CHANGELOG.md?

File: setup.sh [hunk 1]
```diff
@@ -34,111 +64,115 @@ sudo apt-get install -y libusb-1.0-0-dev libreadline-dev libncurses-dev
...
+  I2S_MODULE_CFLAGS="-DRPI_4B"
+  SEP=" "
 fi
 
-if [ $# -ge 1 ] && [ $1 = "xvf3510" ] ; then
-    pushd $RPI_SETUP_DIR/loader/i2s_master > /dev/null
-    I2S_MASTER_FLAG="-DI2S_MASTER"
+case $I2S_MODE in
+  master)
+    I2S_MODULE_CFLAGS="$I2S_MODULE_CFLAGS$EP-DI2S_MASTER"
```
Comment (lucianomartin) [inline setup.sh:74]: A space is needed in I2S_MODULE_CFLAGS when two flags are used

Comment (DaveAtkinsonXMOS) [reply setup.sh]: Ah yes. Fixed.

**Review by lucianomartin — APPROVED** [review]
Looks good

**Pull request merged**